terraform/20-app/cloud-front.front-end.tf

```diff
@@ -180,6 +195,31 @@ resource "aws_cloudfront_origin_request_policy" "front_end" {
   }
 }
 
+resource "aws_cloudfront_origin_request_policy" "front_end_auth" {
+  name = "${local.prefix}-front-end-auth"
+
+  cookies_config {
+    cookie_behavior = "whitelist"
+    cookies {
+      items = [
+        "next-auth.session-token",
+        "__Secure-next-auth.session-token",
+        "next-auth.csrf-token",
+        "__Host-next-auth.csrf-token",
+        "next-auth.callback-url",
+        "next-auth.state"
+      ]
+    }
+  }
+
+  headers_config {
+    header_behavior = "allViewer"
```

Comment:
are allViewers headers necessary or can we whitelist key headers like Authorization, Accept and Content-Type?